README.md

```diff
@@ -2046,3 +2050,24 @@ sequence,hostclass,min_size,desired_size,max_size,instance_type,extra_disk,iops,
 
 In the above pipeline, mhcbar will be integration tested by passing ```mhcbar_integration``` as the argument to the ```test_command``` on the generic ```test_hostclass``` defined in our example ```disco_aws.ini``` above. In contrast, mhcfoo will be integration tested by passing ```mhcfoo_integration``` as the argument to the ```test_command``` on it's specially defined ```test_hostclass```. And because mhcnointegrationtests left the ```integration_test``` column empty, no integration tests will be run for it.
 
+#### Deployment Strategies
+
+##### Classic
+
+Classic deployment strategy is used by default. Instances with a new AMI are spun up in the existing ASG and automatically added to the existing nerve and ELB groups. There are a number of problems with this strategy, so it is slated for deprecation and removal soon. Let us speak no more of it.
+
+##### Blue/Green
+
+Blue/Green deployment strategy is currentlt opt-in only.
+
+The process is as follows:
+
+* A new ASG is created with an 'is_testing' tag and attached to a dedicated 'testing' ELB.
```

Comment:
maybe add a note that clarifies and says for `Blue/Green` strategy, there are two ASGs per hostclass, one for `Green` phase and another for `Blues` phase...